File: FlightSite/FlightSite/ReadModels/FlightRm.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace FlightSite.ReadModels
{
    public record FlightRm(
        Guid Id,
        string Airline,
        string Price,
        TimePlaceRM Departue,
        TimePlaceRM Arrival,
        int RemainingNumberOfSeats
        );

}

File: FlightSite/FlightSite/ClientApp/src/app/api/models/flight-rm.ts
/* tslint:disable */
/* eslint-disable */
import { TimePlaceRm } from './time-place-rm';
export interface FlightRm {
  airline?: null | string;
  arrival?: TimePlaceRm;
  departue?: TimePlaceRm;
  id?: string;
  price?: null | string;
  remainingNumberOfSeats?: number;
}
